### ISSUE TITLE
Rmove CORS headers in production

### ISSUE THREAD

**Issue by axtux** [oSoc19/engie-switch#53]
Example for OPTIONS:
```
add_header 'Access-Control-Allow-Origin' '*';
add_header 'Access-Control-Allow-Methods' 'GET, OPTIONS';
# Custom headers and headers various browsers *should* be OK with but are$
add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-Wit$
# Tell client that this pre-flight info is valid for 1 year
add_header 'Access-Control-Max-Age' 31536000;
return 204;
```
Example for GET:
```
add_header 'Access-Control-Allow-Origin' '*';
add_header 'Access-Control-Allow-Methods' 'GET, OPTIONS';
add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-Wit$
add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
```

**Comment by axtux** [comment]
Those should be disabled on production server. Maybe a config entry to avoid it?

**Comment by axtux** [comment]
CORS added in 206862e6d70425f8fe1088c3274fd7a956c93b40

**Comment by axtux** [comment]
Fixed by 53806b5b90f8066156b3f43792d15ab20707f20e
